static/js/click_handlers.js

```diff
@@ -189,6 +189,40 @@ exports.initialize = function () {
         var local_id = $(this).attr('data-reaction-id');
         var message_id = rows.get_message_id(this);
         reactions.process_reaction_click(message_id, local_id);
+        $(".tooltip").remove();
+    });
+
+    // TOOLTIP FOR MESSAGE REACTIONS
+
+    $('#main_div').on('mouseenter', '.message_reaction', function (e) {
+        e.stopPropagation();
+        var elem = $(this);
```

Comment:
Generally, we try to use `e.currentTarget` rather than `this` in our click handlers.  It's sorta a losing war because everyone is used to `this`, but `this` can have highly unexpected behavior when refactored to inside loops and other constructs.

But in any case, we should certainly be accessing `elem`, not `this`, in the code below.

Reply:
Changed `this` to `e.currentTarget`.